index.bs

```diff
@@ -1080,11 +1080,13 @@ The modules spec should only invoke this operation from methods of Source Text M
 1. Let _deps_ be DependencyGraph(_root_).
 1. For each _dep_ in _deps_, do:
   1. Let _depStageEntry_ be GetCurrentStage(_dep_).
-  1. If _depStageEntry_.[[Stage]] is "link" and _dep_.[[Module]] is a Function object, then:
-    1. Let _f_ be _dep_.[[Module]].
-    1. Let _m_ be ? _f_().
-    1. Set _dep_.[[Module]] to _m_.
-    1. Perform UpgradeToStage(_dep_, "ready").
+  1. If _dep_.[[Module]] is a Function object, then:
```

Comment:
correcting steps (from editorial point of view) to evaluate the factory, and taking into consideration that the factory can only return a namespace exotic object.